README.md

```diff
@@ -77,6 +77,21 @@ passport.use(new wsfedsaml2(
 ));
 ~~~
 
+### Configure strategy for ADFS 

```

Comment:
by default ADFS uses WS-Federation, so the title could be confused

maybe `Configure strategy for ADFS (SAML)`?